charts/fluid/fluid/templates/webhook/webhook.yaml

```diff
@@ -32,6 +32,16 @@ spec:
                   fieldPath: metadata.namespace
             - name: TIME_TRACK
               value: "false"
+            {{- if .Values.webhook.serverlessInject }}
+            {{- if .Values.webhook.serverlessInject.platformLabel }}
```

Comment:
How about naming it `platformKey` and `KEY_SERVERELESS_PLATFORM` ? Just a suggestion. WDYT?

Reply:
Done